tests/test_bess.py

```diff
@@ -22,7 +22,7 @@
     PlaceholderNegativeSampler,
     TripleBasedShardedNegativeSampler,
 )
-from besskge.scoring import ComplEx, TransE
+from besskge.scoring import DistMult, TransE
```

Comment:
Would it be worth unit testing a single fwd execution of the scoring functions against their pykeen counterpart? (Not necessarily in this PR!)

Reply:
Good point, I did want to have some unit tests for scoring functions but wasn't sure how to structure them. Comparing against pykeen might be the most logical way, I agree. I'll have a separate PR for that.